src/app/code/community/Shopgate/Cloudapi/Model/Api2/Wishlists/Items/Rest/Customer/V2.php

```diff
@@ -122,6 +140,29 @@ private function addItemToWishlist(array $data, Mage_Wishlist_Model_Wishlist $wi
         return $wishlist->getData('last_added_item');
     }
 
+    /**
+     * @param array                        $wishlistItemId
+     * @param Mage_Wishlist_Model_Wishlist $wishlist
+     *
+     * @return Mage_Wishlist_Model_Item
+     * @throws Mage_Api2_Exception
+     */
+    private function removeItemsFromWishlist($wishlistItemIds, Mage_Wishlist_Model_Wishlist $wishlist)
+    {
+        try {
+            Mage::dispatchEvent(
+                'shopgate_cloud_api2_wishlists_items_remove',
+                array(
+                    'wishlistItemIds' => $wishlistItemIds,
```

Comment:
I think with the styles, the `=>` should align

Reply:
yes